doc/rtd/reference/user_files.rst

```diff
@@ -0,0 +1,48 @@
+.. _user_files:
+
+User Files: log and configuration
```

Comment:
Can we avoid the use of "user files:" somehow?

Reply:
now that this has its own page that is redundant